modin/logging/config.py

```diff
@@ -100,7 +100,7 @@ def configure_logging():
     logfile = RotatingFileHandler(
         filename=log_filename,
         mode="a",
-        maxBytes=LogMemorySize.get() * int(1e5),
+        maxBytes=int(1e7),
```

Comment:
We make the `LogMemorySize` configurable. I hardcoded the API call logs to default to 10MB per file (same default at the memory logs). I do not think it is necessary to make this configurable as well, but open to other opinions.